### ISSUE TITLE
Cannot build on darwin undefined: unix.SYS_IOCTL

### ISSUE THREAD

**Issue by JWDobken** [hashicorp/terraform-provider-kubernetes#1045]
While following the [developer guide](https://github.com/hashicorp/terraform-provider-kubernetes/blob/master/_about/CONTRIBUTING.md#development-environment) I am trying to build the package for mac:

```cmd
$ GOOS=darwin GOARCH=amd64 make build
==> Checking that code complies with gofmt requirements...
go install
# github.com/moby/term
vendor/github.com/moby/term/tc.go:13:28: undefined: unix.SYS_IOCTL
vendor/github.com/moby/term/tc.go:18:28: undefined: unix.SYS_IOCTL
vendor/github.com/moby/term/termios_bsd.go:24:31: undefined: unix.SYS_IOCTL
vendor/github.com/moby/term/termios_bsd.go:37:31: undefined: unix.SYS_IOCTL
make: *** [build] Error 2
```

- System Version: macOS 10.15.7 (19H2)
- Kernel Version: Darwin 19.6.0
- go version go1.14.10 darwin/amd64

I found a related issue with another project: https://github.com/ory/dockertest/issues/212

**Comment by JWDobken** [comment]
Reproduce the issue with docker:

Run the alpine docker container interactively:
```
$ docker run -ti -e GOOS=darwin -e GOARCH=amd64 -v $GOPATH/bin:/go/bin -v $(PWD):/go/terraform-provider-kubernetes golang:1.14.10-alpine3.12 sh
```

then:

```sh
/go # cd terraform-provider-kubernetes
/go/terraform-provider-kubernetes # go build
# github.com/moby/term
vendor/github.com/moby/term/tc.go:13:28: undefined: unix.SYS_IOCTL
vendor/github.com/moby/term/tc.go:18:28: undefined: unix.SYS_IOCTL
vendor/github.com/moby/term/termios_bsd.go:24:31: undefined: unix.SYS_IOCTL
vendor/github.com/moby/term/termios_bsd.go:37:31: undefined: unix.SYS_IOCTL
```

**Comment by alexsomesan** [comment]
Indeed, this seems to be an issue. It fails building on native macOS as well.
Fix is in PR above ^^

**Comment by alexsomesan** [comment]
We just released v1.13.3 which should no longer have this issue.
